includes/footer

```diff
@@ -16,14 +16,14 @@
       </section>
 
       <section class="col-sm-6 col-md-5ths">
-        <h4><a href="/documentation/">Documentation</a></h4>
+        <h4><a href="/resources/">Resources</a></h4>
         <ul class="shush-list">
           <li><a href="/getting-started/">Getting started</a></li>
           <li><a href="https://doc.perl6.org/">Perl&nbsp;6 Documentation</a></li>
           <li><a href="https://doc.perl6.org/language/5to6">Perl&nbsp;5 to Perl 6</a>,
               an introduction for Perl&nbsp;5 programmers</li>
         </ul>
-          <a href="/documentation/" class="btn btn-success">More</a>
+          <a href="/references/" class="btn btn-success">More</a>
```

Comment:
The `<a href="/references/" class="btn btn-success">More</a>` should have `/resources/` in the link, not `/references/`